Directory.Build.props

```diff
@@ -16,4 +16,12 @@
     <!-- .NET 6+ packages support back to API-21 -->
     <SupportedOSPlatformVersion>21</SupportedOSPlatformVersion>
   </PropertyGroup>
+  
+  <!-- Folders that .targets files need to go into -->
+  <ItemGroup>
+    <NuGetTargetFolders Include="build\monoandroid90" />
+    <NuGetTargetFolders Include="build\net6.0-android31.0" />
+    <NuGetTargetFolders Include="buildTransitive\monoandroid90" />
+    <NuGetTargetFolders Include="buildTransitive\net6.0-android31.0" />
```

Comment:
I would pick a name that more likely won't collide with anything, like `@(AndroidXNuGetTargetFolders)`.

Reply:
👍